.eslintrc.js

```diff
@@ -16,5 +16,11 @@ module.exports = {
   },
   rules: {
     
+    "indent": 0,
```

Comment:
Some of these rules you are ignoring aren't allowing you to write standard well-linted javascript.

Please remove the `indent` rule, `no-restricted-globals` and `camelcase` rules